datasource/src/main/scala/quasar/blobstore/azure/AzureBlobstore.scala

```diff
@@ -53,7 +52,16 @@ class AzureBlobstore[F[_]: ConcurrentEffect: MonadResourceErr: RaiseThrowable](
       }
   }
 
-  def isResource(path: ResourcePath): F[Boolean] = errorNotImplemented
+  def isResource(path: ResourcePath): F[Boolean] = {
+    val res: F[Boolean] = for {
+      single <- download(pathToAzurePath(path), new BlobRange().withCount(java.lang.Long.valueOf(0L)))
```

Comment:
Did you consider `BlobURL#getProperties()`? It seems like it might be a more declarative means of achieving this, but I'm not sure if they're equivalent.

Reply:
I haven't - will give it a try

Reply:
@wemrysi I changed it to `BlobURL#getProperties()` - seems to work fine